airbyte-integrations/connectors/source-retently/source_retently/schemas/nps.json

```diff
@@ -1,12 +1,35 @@
 {
-    "score": { "type": "integer" },
-    "scoreSum": { "type": "integer" },
-    "metricsType": { "type": "string" },
-    "promoters": { "type": "integer" },
-    "passives": { "type": "integer" },
-    "detractors": { "type": "integer" },
-    "promotersCount": { "type": "integer" },
-    "passivesCount": { "type": "integer" },
-    "detractorsCount": { "type": "integer" },
-    "totalResponses": { "type": "integer" }
+  "type": "object",
```

Comment:
Add `additionalProperties: true`

Reply:
Any reason why we would need to add `additionalProperties: true`?

Reply:
Gotcha, that's good to know, does that count for a stream that initially had an invalid json schema?